Documentation/MRTKNuGetPackage.md

```diff
@@ -0,0 +1,66 @@
+# Mixed Reality Toolkit NuGet Package
+
+Mixed Reality Toolkit (MRTK) is now available as a NuGet package on NuGet.org. There are some differences when it comes to consuming NuGet version of MRTK as opposed to a .unitypackage, read [NuGet Package Considerations](###%20NuGet%20Package%20Considerations) below. If any issues are encountered, file an issue using this [template](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/new?assignees=&labels=Bug,Package%20Management%20-%20NuGet&template=bug-report.md&title=).
+
+## Installing the NuGet Package
+
+Follow these instructions to add the Mixed Reality Toolkit as a NuGet package to your project.
+
+1. Download the latest [NuGetForUnity](https://github.com/GlitchEnzo/NuGetForUnity/releases/latest) .unitypackage
+1. Import the package into your Unity project, [instructions](https://docs.unity3d.com/Manual/AssetPackages.html).
+1. Configure NuGet For Unity, click on the menu **Edit > Preferences...**.
+    - On the left, find **NuGet for Unity** tab.
+    - Un-check **Read-Only Package Files**.
+1. In the Unity menu bar, click on **NuGet > Manage NuGet Packages**.
+1. In the Search box, enter `Microsoft.MixedReality.Toolkit`, and choose the package you would like to install.
+    - **Microsoft.MixedReality.Toolkit.Foundation** – The core package for MRTK.
+    - **Microsoft.MixedReality.Toolkit.Examples** – The package that contains all of our examples.
+    - **Microsoft.MixedReality.Toolkit.Extensions** – The package that contains extensions services and/or data providers. (more information).
+    - **Microsoft.MixedReality.Toolkit.Tools** – Contains some of the tooling that comes with MRTK (Build Window, etc).
+
+### Updating MRTK NuGet Packages
+
+Steps 1-3 above will only need to be done once for your project, and the update is a much simpler step. Once newer packages available on NuGet.org (including prerelease), follow these steps:
+
+1. In the Unity menu bar, click on NuGet > Manage NuGet Packages.
+1. Switch to the **Updates** tab.
+    - Check the **Show prerelease** box if you want to get latest prerelease version.
+1. Update the packages desired.
+
+## NuGet Package Considerations
+
+The release of MRTK as NuGet package is a new delivery mechanism being explored and there are a couple of key benefits and considerations one must make when choosing whether to consume the NuGet version of MRTK.
+
+### Upgrading to NuGet from .unitypackage or Source
+
+NuGet package consists of compiled binaries as opposed to loose script files, and as such, the C# script asset identifiers are different. As such, the assets like prefabs in the MRTK package have been updated to reference the appropriate script. A project using the .unitypcakge or Source version of MRTK will have to do the same, and although there is code for it this is not a supported scenario, yet.
```

Comment:
It's a bit unclear what the impact of this is on users. Is it something like "If you are referencing mrtk scripts monobehaviours from a .unitypackage in your project, you may need to resetup these references when moving to nuget."? I think that this is what the documentation means, but its not super clear.

Also, if there is an associated cost like this, we should call out the benefit of doing so. I think you should call out that there will be strong version control with an easier update model. This might be inferred reading through the different items, but it could also be specifically called out.

Reply:
ah the compiled binaries vs source files below is awesome too.

Reply:
Rewording.

Reply:
"although there is code for this" 

Should we provide the code in an experimental namespace? If not, we should just say that it isn't supported.